docs/en_US/faq.md

```diff
@@ -12,9 +12,9 @@ here](https://www.eff.org/https-everywhere/development).
     network?](#why-is-https-everywhere-preventing-me-from-joining-this-hotelschoolother-wireless-network)
 *   [Will there be a version of HTTPS Everywhere for IE, Safari, or some other
     browser?](#will-there-be-a-version-of-https-everywhere-for-ie-safari-or-some-other-browser)
-*   [Why use a whitelist of sites that support HTTPS? Why can't you try to use
+*   [Why use a allowlist of sites that support HTTPS? Why can't you try to use
```

Comment:
I find this question in the FAQ is misleading on how HTTPS Everywhere works. It is also inconsistent with the terminology we use. I believe this questions would be better rephrased to avoid confusion.

HTTPS Everywhere use a collection of peer-reviewed rulesets to upgrade insecure HTTP connections to HTTPS. We also use the terminology "Add a rule for this site" on the extension user interface, our users "Enable"/ "Disable" rules with checkbox as needed. 

Would something like `Why use bundled rulesets to upgrade insecure HTTPS connections?` better? 

P.S. I am not a native English speaker, so please forgive me if I am asking a weird question.

Reply:
I would suggest "Why not just redirect all websites by default?". The simplest wording possible is often the best one.

Reply:
I can change the wording, just have to coordinate a small effort to get the translation. I can schedule that into the next round of documentation updates. Wanted to focus on this renaming portion first. I will note this in the documentation items that need to be done.

Thank you both so much for reviewing with great care and compassion.